v1.0/create-security-certificates.md

```diff
@@ -131,54 +153,48 @@ If you need to troubleshoot this command's behavior, you can change its [logging
     ~~~
 
     ~~~
-    -rw-r--r--  1 maxroach  maxroach  1.1K Apr 19 09:35 ca.crt
-    -rw-r--r--  1 maxroach  maxroach  1.2K Apr 19 09:36 node.crt
-    -rw-------  1 maxroach  maxroach  1.6K Apr 19 09:36 node.key
+    total 40
+    -rw-r--r--  1 maxroach  maxroach  1.1K Jul 10 14:12 ca.crt
+    -rw-r--r--  1 maxroach  maxroach  1.1K Jul 10 14:13 client.maxroach.crt
+    -rw-------  1 maxroach  maxroach  1.6K Jul 10 14:13 client.maxroach.key
+    -rw-r--r--  1 maxroach  maxroach  1.2K Jul 10 14:16 node.crt
+    -rw-------  1 maxroach  maxroach  1.6K Jul 10 14:16 node.key
     ~~~
 
 2. Upload the files to the first node.
 
-3. Create the certificate and key for the second node, using the `--overwrite` flag to replace the files created for the first node:
+3. Delete the local copy of the first node's certificate and key:
+
+    ~~~ shell
+    $ rm -rf certs/node.crt certs/node.key
```

Comment:
you can drop the `-rf`. We're definitely not deleting directories (`-r`) and the permissions set by `cockroach cert` means we don't need to force the delete (`-f`).